cmd/api/src/database/migration/migrations/v5.16.0.sql

```diff
@@ -19,3 +19,11 @@ CREATE EXTENSION IF NOT EXISTS pg_trgm;
 
 CREATE INDEX IF NOT EXISTS idx_saved_queries_description ON saved_queries using gin(description gin_trgm_ops);
 CREATE INDEX IF NOT EXISTS idx_saved_queries_name ON saved_queries USING gin(name gin_trgm_ops);
+
+VALUES (3, 
+'analysis.citrix_rdp_support', 
+'Citrix RDP Support', 
+'This configuration parameter toggles Citrix support during post-processing. When enabled, computers identified with a ''Direct Access Users''', 
```

Comment:
This needs to be updated with the new copy

Reply:
Additionally the INSERT statement is missing and we don't want to insert a specified id of 3 but rely on the autoincr sequence